src/styles/blocks/body.scss

```diff
@@ -0,0 +1,4 @@
+body {
+  margin: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
```

Comment:
Why do you need this file if you set the styles for a body in general.scss file ?